source/wait_time.c

```diff
@@ -0,0 +1,27 @@
+#ifndef STRUCTS
+#define STRUCTS
+
+#include "../headers/structs.h"
+
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+
+/*Checking if passengers can finish*/
+void wait_time(transition_system *t) {
+    int i, j , p_W;
+    passenger p;
+    for(i = 0; i < t->length; i++) {
+        p = t->passengers[i];
```

Comment:
Jeg vil argumentere for, at jeres if-condition er så simpel, at det ikke er nødvendigt med en ekstra lokal variabel.
Den er blevet brugt i andre source-filer, fordi udtrykket i if-statementet simpelthen blev for langt.
```suggestion
```